Toolchain/BuildClang.sh

```diff
@@ -225,177 +221,112 @@ popd
+            -DRUNTIMES_x86_64-pc-serenity_CMAKE_BUILD_TYPE="Release" \
+            -DRUNTIMES_i386-pc-serenity_CMAKE_SYSROOT="$BUILD/i686/Root" \
+            -DRUNTIMES_x86_64-pc-serenity_CMAKE_SYSROOT="$BUILD/x86_64/Root" \
+            -DRUNTIMES_i386-pc-serenity_LIBCXX_ENABLE_LOCALIZATION=OFF \
+            -DRUNTIMES_x86_64-pc-serenity_LIBCXX_ENABLE_LOCALIZATION=OFF \
+            -DRUNTIMES_i386-pc-serenity_LIBCXX_ENABLE_FILESYSTEM=OFF \
+            -DRUNTIMES_x86_64-pc-serenity_LIBCXX_ENABLE_FILESYSTEM=OFF \
+            -DRUNTIMES_i386-pc-serenity_COMPILER_RT_OS_DIR="serenity" \
+            -DRUNTIMES_x86_64-pc-serenity_COMPILER_RT_OS_DIR="serenity" \
+            -DRUNTIMES_i386-pc-serenity_COMPILER_RT_BUILD_BUILTINS=ON \
+            -DRUNTIMES_x86_64-pc-serenity_COMPILER_RT_BUILD_BUILTINS=ON \
+            -DRUNTIMES_i386-pc-serenity_COMPILER_RT_EXCLUDE_ATOMIC_BUILTIN=OFF \
+            -DRUNTIMES_x86_64-pc-serenity_COMPILER_RT_EXCLUDE_ATOMIC_BUILTIN=OFF \
+            -DRUNTIMES_i386-pc-serenity_COMPILER_RT_BUILD_CRT=ON \
+            -DRUNTIMES_x86_64-pc-serenity_COMPILER_RT_BUILD_CRT=ON \
+            -DRUNTIMES_i386-pc-serenity_COMPILER_RT_BUILD_SANITIZERS=OFF \
+            -DRUNTIMES_x86_64-pc-serenity_COMPILER_RT_BUILD_SANITIZERS=OFF \
+            -DRUNTIMES_i386-pc-serenity_COMPILER_RT_BUILD_LIBFUZZER=OFF \
+            -DRUNTIMES_x86_64-pc-serenity_COMPILER_RT_BUILD_LIBFUZZER=OFF \
+            -DRUNTIMES_i386-pc-serenity_COMPILER_RT_BUILD_MEMPROF=OFF \
+            -DRUNTIMES_x86_64-pc-serenity_COMPILER_RT_BUILD_MEMPROF=OFF \
+            -DRUNTIMES_i386-pc-serenity_COMPILER_RT_BUILD_PROFILE=OFF \
+            -DRUNTIMES_x86_64-pc-serenity_COMPILER_RT_BUILD_PROFILE=OFF \
+            -DRUNTIMES_i386-pc-serenity_COMPILER_RT_BUILD_XRAY=OFF \
+            -DRUNTIMES_x86_64-pc-serenity_COMPILER_RT_BUILD_XRAY=OFF \
+            -DRUNTIMES_i386-pc-serenity_COMPILER_RT_BUILD_ORC=OFF \
+            -DRUNTIMES_x86_64-pc-serenity_COMPILER_RT_BUILD_ORC=OFF \
+            -DRUNTIMES_i386-pc-serenity_CMAKE_C_FLAGS="-march=i686 -ftls-model=initial-exec -nostdlib -nostdlib++" \
+            -DRUNTIMES_x86_64-pc-serenity_CMAKE_C_FLAGS="-march=x86-64 -ftls-model=initial-exec -nostdlib -nostdlib++" \
+            -DRUNTIMES_i386-pc-serenity_CMAKE_CXX_FLAGS="-march=i686 -ftls-model=initial-exec -nostdlib -nostdlib++" \
+            -DRUNTIMES_x86_64-pc-serenity_CMAKE_CXX_FLAGS="-march=x86-64 -ftls-model=initial-exec -nostdlib -nostdlib++" \
+            -DBUILTINS_i386-pc-serenity_CMAKE_BUILD_TYPE="Release" \
+            -DBUILTINS_x86_64-pc-serenity_CMAKE_BUILD_TYPE="Release" \
+            -DBUILTINS_i386-pc-serenity_CMAKE_SYSROOT="$BUILD/i686/Root" \
+            -DBUILTINS_x86_64-pc-serenity_CMAKE_SYSROOT="$BUILD/x86_64/Root" \
+            -DLLVM_ENABLE_BINDINGS=OFF \
             -DLLVM_INCLUDE_BENCHMARKS=OFF \
             -DLLVM_INCLUDE_TESTS=OFF \
             -DLLVM_LLVM_BUILD_LLVM_DYLIB=ON \
```

Comment:
This option is unrecognized by the llvm cmake infrastructure. Should it be `LLVM_BUILD_LLVM_DYLIB` instead of `DLLVM_LLVM_BUILD_LLVM_DYLIB` ?